test/Microsoft.AspNetCore.Hosting.Tests/WebHostTests.cs

```diff
@@ -67,11 +67,13 @@ public void UseStartupThrowsWithNull()
         }
 
         [Fact]
-        public void NoDefaultAddressesIfNotConfigured()
+        public void NoDefaultAddressesAndDoNotPreferHostingUrlsIfNotConfigured()
```

Comment:
This is just making sure the default is false. I will add the check for the new flag in the other tests where only address is configured.